client/src/common/components/Members/index.jsx

```diff
@@ -128,24 +131,31 @@ class MembersBox extends PureComponent {
   };
 
   renderMemberList = () => {
-    const { members } = this.props;
+    const {
+      intl: { formatMessage },
+      members
```

Comment:
will users who have deleted their accounts be members of the list or cohort? I think not. So usage of format message is not necessary for all members' components. We have to protect those places where the user name has to be populated: item, activities. On profile view (User who delete thier account won't display thier profile :-), on list/cohort members we won't display users who deleted their accounts. I somebody will delete thier account they will be removed from lists or cohorts.